<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MitkDrawingTool</class>
 <widget class="QWidget" name="MitkDrawingTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
    <widget class="ActionPushButton" name="newLabelPushBtn">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Add New Label</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
    <widget class="ActionPushButton" name="createMaskPushBtn">
     <property name="text">
      <string>Create mask</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QmitkLabelSetWidget" name="labelSetWidget" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="toolGUIArea" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </widget>
   </item>
   <item>
    <widget class="QmitkToolSelectionBox" name="toolSelectionBox" native="true"/>
   </item>
   <item>
    <widget class="QPushButton" name="ConfirmSegBtn">
     <property name="styleSheet">
      <string notr="true">QPushButton { font:21px; height:30px;width:150px;color:black; background-color:rgb(50,220,0); border-style: solid; border-color: #9f0000; border-width: 1px; border-radius: 5px; padding: 3px;}
QPushButton:checked{ background-color: black; border-style: outset; border-style: solid; border-color: black; border-width: 1px; border-radius: 5px;}
QPushButton:hover{ background-color: rgb(0,200,57);border-style: outset; border-style: solid; border-color: black; border-width: 1px; border-radius: 5px;}
QPushButton:pressed{ background-color: rgb(0,100,0);border-style: outset; border-style: solid; border-color: black; border-width: 1px; border-radius: 5px;}                 </string>
     </property>
     <property name="text">
      <string>Confirm Mask</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkToolSelectionBox</class>
   <extends>QWidget</extends>
   <header location="global">QmitkToolSelectionBox.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkLabelSetWidget</class>
   <extends>QWidget</extends>
   <header location="global">Qmitk/QmitkLabelSetWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ActionPushButton</class>
   <extends>QPushButton</extends>
   <header>ActionPushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
